### PR TITLE
[TACHYON-1428] Documented parts of the tests of the tachyon.client.file package in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 1]
```diff
@@ -107,6 +124,11 @@ public void createExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#delete(TachyonFile, DeleteOptions)} method.
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 2]
```diff
@@ -126,6 +153,11 @@ public void deleteExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#free(TachyonFile, FreeOptions)} method.
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 3]
```diff
@@ -145,6 +182,11 @@ public void freeExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#getInfo(TachyonFile, GetInfoOptions)} method.
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 4]
```diff
@@ -166,6 +213,12 @@ public void getInfoExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#listStatus(TachyonFile, ListStatusOptions)}
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 5]
```diff
@@ -187,6 +245,12 @@ public void listStatusExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#loadMetadata(TachyonURI, LoadMetadataOptions)}
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 6]
```diff
@@ -210,6 +279,11 @@ public void loadMetadataExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#mkdir(TachyonURI, MkdirOptions)} method.
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 7]
```diff
@@ -230,6 +309,12 @@ public void mkdirExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#mount(TachyonURI, TachyonURI, MountOptions)}
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 8]
```diff
@@ -254,6 +344,12 @@ public void mountExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#open(TachyonURI, OpenOptions)} method to complete
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 9]
```diff
@@ -262,6 +358,12 @@ public void openSuccessTest() throws Exception {
     Mockito.verify(mFileSystemMasterClient).getFileId("/");
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#open(TachyonURI, OpenOptions)} method to fail
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 10]
```diff
@@ -286,6 +393,12 @@ public void openExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#openIfExists(TachyonURI, OpenOptions)} method to
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 11]
```diff
@@ -295,6 +408,12 @@ public void openIfExistsSucessTest() throws Exception {
     Mockito.verify(mFileSystemMasterClient).getFileId("/");
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#openIfExists(TachyonURI, OpenOptions)} method to
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 12]
```diff
@@ -303,6 +422,12 @@ public void openIfExistsFailTest() throws Exception {
     Mockito.verify(mFileSystemMasterClient).getFileId("/");
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#rename(TachyonFile, TachyonURI, RenameOptions)}
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 13]
```diff
@@ -324,6 +454,11 @@ public void renameExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#setState(TachyonFile, SetStateOptions)} method.
+   *
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

File: clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java [hunk 14]
```diff
@@ -343,6 +483,11 @@ public void setStateExceptionTest() throws Exception {
     }
   }
 
+  /**
+   * Test for the {@link AbstractTachyonFileSystem#unmount(TachyonURI, UnmountOptions)} method.
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java]: Test->Tests

**Comment by yupeng9** [comment]
Thanks for the doc improvement. I left some comment.

**Comment by HorizonNet** [comment]
@yupeng9 Thanks for the comments. I updated the PR accordingly.

**Comment by HorizonNet** [comment]
Jenkins, test this please.

**Comment by calvinjia** [comment]
LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**